src/main/java/org/myrobotlab/service/Runtime.java

```diff
@@ -436,9 +434,8 @@ synchronized private static Map<String, ServiceInterface> createServicesFromPlan
         runtime.error("could not get %s from plan", service);
         continue;
       }
-      sc.state = "CREATING";
+      // sc.state = "CREATING";

```

Comment:
If we don't need this anymore, can we entirely remove this line?

Reply:
absolutely ... :+1: